src/d3d8/d3d8_device_child.h

```diff
@@ -32,6 +32,10 @@ namespace dxvk {
     }
     
     ULONG STDMETHODCALLTYPE Release() {
+      // ignore Release calls on objects with 0 refCount
+      if(unlikely(!this->m_refCount))
+        return this->m_refCount;
+

```

Comment:
This seems good. This behavior was supposed to come from `ComObjectClamp` before it was overridden here.

Reply:
This is only needed to prevent a regression with Motor City Online and this PR, since the game will call Release() even on objects with 0 recount and won't stop calling Release() until it gets a 0 refcount again (going through uint32 values basically, as it underruns) . I haven't seen anything else rely on it tbh.